reference/reference-environment-variables.md

```diff
@@ -435,6 +435,21 @@ So you can alter the build&start process for your application.
 |ES_ADDON_VERSION | ElasticSearch Version | 7 | X  |
 {{< /table >}}
 
+### Pulsar
+
+{{<table "table table- bordered" "text-align:center" >}}
+| <center>Name</center>    | <center>Description</center> | <center>Default value</center> | <center>Read Only</center> |
+| ------------------------ | ----------------------------------------- | ------------------------------ | -------------------------- |
+| ADDON_PULSAR_HOSTNAME    | The host to connect to the Pulsar Cluster | Generated upon creation | X |
```

Comment:
Don't we want to keep the list alphabetically sorted? Other than that, ok!

Reply:
Fixed